Packs/RecordedFutureASI/pack_metadata.json

```diff
@@ -3,7 +3,7 @@
     "prevName": "Recorded Future ASI",
     "description": "Helps you take risk prioritization to the next level by helping you identify the biggest weaknesses within your attack surface.",
     "support": "partner",
-    "currentVersion": "2.0.9",
```

Comment:
This is not really a big change, please change the version to 2.0.10. (in the release note and in the pack metadata.json)